boards/posix/native_posix/native_posix.dts

```diff
@@ -91,6 +92,16 @@
 		label = "I2C_0";
 	};
 
+	spi0: spi@200 {
```

Comment:
Does the emulator need a base memory address (of 200)? If not, can we drop it?

Reply:
No it doesn't need it, but peripherals are supposed to be at a particular address - see I2C. That said I don't think it is essential so I'll drop it.

Reply:
Hmm actually I can't, since the SPI binding requires it.

Reply:
No worries, then. We can leave it